doc/source/yt4differences.rst

```diff
@@ -372,5 +384,24 @@ and load them into a yt session, but now this occurs from within a python
 session.  For more information see:
 :ref:`Loading Sample Data <loading-sample-data>`
 
-API Changes
------------
+Some Widgets
+^^^^^^^^^^^^
+
+In yt, we now have some simple display wrappers for objects if you are running
+in a Jupyter environment with the `ipywidgets
+<https://ipywidgets.readthedocs.io/>`_ package installed.  For instance, the
+``ds.fields`` object will now display field information in an interactive
+widget, and three-element unyt arrays (such as ``ds.domain_left_edge``) will be
+displayed interactively as well.
+
+The package `widgyts <https://widgyts.readthedocs.io>`_ provides interactive,
+yt-specific visualization of slices, projections, and additional dataset display
+information.
+
+New External Packages
+^^^^^^^^^^^^^^^^^^^^^
+
+As noted above (:ref:`yt-units-is-now-unyt`), ``unyt`` has been extracted from
```

Comment:
For consistency (since we don't usually bartick yt itself)
```suggestion
As noted above (:ref:`yt-units-is-now-unyt`), unyt has been extracted from
```